packages/pangomm.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Pangomm < Package
+  description 'pangomm is the official C++ interface for the Pango font layout library.'
+  homepage 'https://developer.gnome.org/pangomm/stable/'
+  version '2.40.1'
+  source_url 'http://ftp.acc.umu.se/pub/GNOME/sources/pangomm/2.40/pangomm-2.40.1.tar.xz'
```

Comment:
We probably shouldn't use a mirror url for the source.  The official project has a newer version available?  https://ftp.gnome.org/pub/GNOME/sources/pangomm/2.41/pangomm-2.41.5.tar.xz

Reply:
I prefer stable version. the latest one requires gcc-7.3 to compile.